torchgeo/datasets/oscd.py

```diff
@@ -0,0 +1,234 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""OSCD dataset."""
+
+import os
+import glob
+import tifffile
+from typing import Callable, Dict, List, Optional
+
+import torch
+import numpy as np
+from PIL import Image
+from torch import Tensor
+
+from .geo import VisionDataset
+from .utils import download_url, extract_archive
+
+
+def sort_bands(x: str) -> str:
```

Comment:
It might be easier to import the function from BigEarthNet instead of duplicating it.

Reply:
We may want to pull the BigEarthNet fn into `torchgeo.datasets.utils` and call it `sort_sentinel2_bands` to make it obvious. We are going to encounter this in any Sentinel-2 dataset which just downloads the raw bands into separate files which will be common.

Reply:
+1 (this is how S2 data is distributed)

Reply:
Submitted a PR that does this: https://github.com/microsoft/torchgeo/pull/237/files
Let me know if it's good!